build.githubactions.ps1

```diff
@@ -128,11 +131,15 @@ function Compile {
 }
 
 function Pack {
-    if (-not $PackageName){
+    if ([string]::IsNullOrWhiteSpace($PackageName) -and [string]::IsNullOrWhiteSpace($NuspecFilePath)){
         Invoke-Execute {
             dotnet pack $ProjectFile -c $Configuration --output $packageOutput --no-build --verbosity normal -p:VersionPrefix=$version -p:NoWarn=NU5123
         }
-    } else {
+    }
+    if ($NuspecFilePath -Like "*.nuspec" -and $PackageName  -Like "EdFi.Suite3.Ods.Extensions.**"){   
```

Comment:
Since this is a build script specific to the extensions repo, in the extensions repo, do we need to be doing this check for the package name being an extensions package on this if statement?

Reply:
I added $PackageName -ne $null